src/components/CodeAssignmentModal/index.jsx

```diff
@@ -285,6 +300,15 @@ export class BaseCodeAssignmentModal extends React.Component {
       };
     }
 
+    let usersDetailResponse = [];
+    try {
+      const response = await LmsApiService.fetchUserDetailsFromEmail(options.emails);
+      usersDetailResponse = response.data;
+    } catch (error) {
+      logError(error);
+    }
+    options.users_details = this.getCleanedUsersDetails(options.emails, usersDetailResponse);
```

Comment:
We need to rename it to `options.user_details`

Reply:
I think the user should be plural here as we are having multiple user data in this array.